selectAllOnClick.js

```diff
@@ -1,27 +0,0 @@
-angular.module('PathOfDamage')
```

Comment:
yeah, you added it to js/ folder, but didn't remove it from the project root directory